.travis.yml

```diff
@@ -7,3 +7,12 @@ node_js:
 - "9"
 addons:
   chrome: stable
+env:
+  - NPM_SCRIPT=test:browserless
+matrix:
+  include:
+    # Run browser tests only on one Node.js version to save time.
+    - node_js: "8"
+      env: NPM_SCRIPT=test
+script:
+  - npm run-script $NPM_SCRIPT
```

Comment:
Might want to use `npm run` here for consistency with the other callers.

Reply:
Done.